mycore-solr/src/main/resources/xsl/solr-xeditor-templates.xsl

```diff
@@ -2,6 +2,10 @@
 <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xed="http://www.mycore.de/xeditor"
   xmlns:mcrsolr="http://www.mycore.de/components/solr" xmlns:mcrxml="xalan://org.mycore.common.xml.MCRXMLFunctions">
   <xsl:param name="MCR.Solr.ServerURL" />
+  <xsl:param name="MCR.Solr.Core.main.Name" />
+  <xsl:param name="MCR.Solr.Core.main.ServerURL" />
+  <xsl:param name="MCR.Solr.Core.classification.Name" />
+  <xsl:param name="MCR.Solr.Core.classification.ServerURL" />
```

Comment:
Now searchmask is flexible but this template still only works for "classification" or "main". This code have also to be that flexible or we can switch back to drop down menu in xed.